Marlin/src/lcd/menu/menu_tramming.cpp

```diff
@@ -43,6 +43,10 @@ float z_measured[G35_PROBE_COUNT] = { 0 };
 static uint8_t tram_index = 0;
 
 bool probe_single_point() {
+  // In BLTOUCH HS mode, the probe travels in a deployed state.
+  // Users of Tramming Wizard might have a badly misaligned bed, so raise Z by the
+  // length of the deployed pin (BLTOUCH stroke < 7mm)
+  current_position.z = (Z_CLEARANCE_BETWEEN_PROBES) + (7 * ENABLED(BLTOUCH_HS_MODE));
```

Comment:
This should be an explicit move such as `do_blocking_move_to_z(..., MMM_TO_MMS(Z_PROBE_SPEED_FAST))`.

Reply:
I just copied it from G35.cpp

Reply:
Is this 7mm clearance used somewhere else? Does that not already exist in other probe routines?

I'd like to see it be a shared constant somewhere rather than repeated in multiple places.